akash1/HTML/ASSIn1/assignment1.Q3.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Q3</title>
+</head>
+<body>
+    <h1>HTML</h1>
+    
```

Comment:
why there is same code in all files with different title only otherwise all 3 files are same?

Reply:
No mam these files are different. As per assignment given by pawan sir. i added those things like i added in assignment1.Q3 <hr> tag. so please recheck the output as per assignment.